src/Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions/IConnectionInformation.cs

```diff
@@ -12,9 +12,10 @@ public interface IConnectionInformation
         IPEndPoint LocalEndPoint { get; }
 
         PipeFactory PipeFactory { get; }
+        IScheduler InputReaderScheduler { get; }
```

Comment:
Is `ThreadPoolDispatching` in `InternalKestrelServerOptions`

Also avoiding double dispatch could be baked into the threadpool instead e.g. https://github.com/aspnet/KestrelHttpServer/pull/1626/files